lib/config/options/index.ts

```diff
@@ -2634,6 +2634,14 @@ const options: RenovateOptions[] = [
     globalOnly: true,
     default: [],
   },
+  {
+    name: 'branchSummaryExtended',
```

Comment:
Do we need to make this configurable? I think branch summary is only consumed by Mend today, and this feature is also for Mend

Reply:
yes i thought to make it in a different flag on its own, because there's a-lot of input needed for the dependency dashboard

i could also print the dependency dashboard itself in a flag if possible, its the same to me